src/wovnio/html/HtmlConverter.php

```diff
@@ -196,13 +196,11 @@ private function buildSnippetCode($adds_backend_error_mark)
         $url_pattern = $this->store->settings['url_pattern_name'];
         $lang_param_name = $this->store->settings['lang_param_name'];
         $lang_code_aliases_json = json_encode($this->store->settings['custom_lang_aliases']);
+        $widget_url = $this->store->settings['widget_url'];
+        $fallback_mark = $adds_backend_error_mark ? 'data-wovnio-type="fallback_snippet"' : '';
         $data_wovnio = htmlentities("key=$token&backend=true&currentLang=$current_lang&defaultLang=$default_lang&urlPattern=$url_pattern&langCodeAliases=$lang_code_aliases_json&langParamName=$lang_param_name&version=WOVN.php");
 
-        if ($adds_backend_error_mark) {
-            return "<script src=\"//j.wovn.io/1\" data-wovnio=\"$data_wovnio\" data-wovnio-type=\"fallback_snippet\" async></script>";
-        } else {
-            return "<script src=\"//j.wovn.io/1\" data-wovnio=\"$data_wovnio\" async></script>";
-        }
+        return "<script src=\"$widget_url\" data-wovnio=\"$data_wovnio\" $fallback_mark async></script>";
```

Comment:
i'm curious where the extra space is coming from in the tests. It doesn't look like there should be a space here

Reply:
Extra space comes from both sides of `$fallback_mark` when `$fallback_mark` is empty.
I didn't want to write similar snippet twice.
Do you think I should change as before?

Reply:
Getting rid of the extra space would be nice but not a big deal.
Maybe you could set `$fallback_mark = $adds_backend_error_mark ? ' data-wovnio-type="fallback_snippet"' : '';` to have a space on the front.
and do this?
`return "<script src=\"$widget_url\" data-wovnio=\"$data_wovnio\"$fallback_mark async></script>";`

Reply:
OK. I will do that.